logic/src/commonMain/kotlin/com/wire/kalium/logic/configuration/UserConfigRepository.kt

```diff
@@ -200,13 +202,20 @@ class UserConfigDataSource(
     }
 
     override fun isReadReceiptsEnabled(): Flow<Either<StorageFailure, Boolean>> =
-        userConfigStorage.isReadReceiptsEnabled().wrapStorageRequest()
+        userConfigStorage.areReadReceiptsEnabled().wrapStorageRequest()
 
     override fun setReadReceiptsStatus(enabled: Boolean): Either<StorageFailure, Unit> =
         wrapStorageRequest {
             userConfigStorage.persistReadReceipts(enabled)
         }
 
+    override fun isTypingIndicatorEnabled(): Flow<Either<StorageFailure, Boolean>> =
+        userConfigStorage.isTypingIndicatorEnabled().wrapStorageRequest()
+
+    override fun setTypingIndicatorStatus(isTyping: Boolean): Either<StorageFailure, Unit> = wrapStorageRequest {
```

Comment:
```suggestion
    override fun setTypingIndicatorStatus(enabled: Boolean): Either<StorageFailure, Unit> = wrapStorageRequest {
```

Or maybe `isTypingEnabled` might be a proper name here, what do you think ?

Reply:
Good catch. I think I named it wrongly as I misunderstood the task at the beginning. Changing it to `isEnabled` to keep coherence with other properties.